tapdb/universe.go

```diff
@@ -405,7 +407,7 @@ func universeUpsertProofLeaf(ctx context.Context, dbTx BaseUniverseStore,
 	// root ID that we'll use to insert the universe leaf overlay.
 	universeRootID, err := dbTx.UpsertUniverseRoot(ctx, NewUniverseRoot{
 		NamespaceRoot: namespace,
-		AssetID:       fn.ByteSlice(leaf.ID()),
+		AssetID:       assetIDBytes,
```

Comment:
I think this is needed for the existing queries to work, as this is how we link it to a genesis asset.

Reply:
Yes, you are right. Removed this commit again and made sure we only use either value in `multiverse_leaves`.